arches/app/templates/views/components/search/advanced-search.htm

```diff
@@ -116,29 +116,48 @@
                 <!-- ko if: cards().length > 0  -->
                 <!-- ko let: {uid: Math.random().toString()} -->
                 <div tabindex="0" role="button" class="search-facet-item disabled" data-bind="onEnterkeyClick, onSpaceClick, 
-                    click: function () { collapsed(!collapsed()) },
-                    attr: {
-                        'aria-expanded': (!collapsed()).toString(),
-                        'aria-controls': uid,
-                        'aria-label': $root.translations.resourceTypeSearchFacets(name),
-                    }
+                click: function () { collapsed(!collapsed()) },
+                attr: {
+                    'aria-expanded': (!collapsed()).toString(),
+                    'aria-controls': uid,
+                    'aria-label': $root.translations.resourceTypeSearchFacets(name),
+                }
                 ">
-                    <div class="search-facet-item-heading">
-                        <strong data-bind="text: name"></strong>
-                        <span><i class="fa report-expander print-hide" data-bind="css: {'fa-angle-down': !collapsed(), 'fa-angle-right': collapsed()}"></i></span>
+                <div class="search-facet-item-heading">
+                    <strong data-bind="text: name"></strong>
+                    <span><i class="fa report-expander print-hide" data-bind="css: {'fa-angle-down': !collapsed(), 'fa-angle-right': collapsed()}"></i></span>
```

Comment:
I don't see any changes in lines 119 to 128. It looks like just a tab removal. Just to clean the PR a bit, could they be tabbed back?